libmachine/mcnutils/b2d.go

```diff
@@ -20,12 +19,8 @@ var (
 	GithubApiToken string
 )
 
-const (
-	timeout = time.Second * 5
-)
-
 func defaultTimeout(network, addr string) (net.Conn, error) {
-	return net.DialTimeout(network, addr, timeout)
+	return net.Dial(network, addr)
```

Comment:
I think it's just a timeout for the `Dial` though, right?  i.e. the initial call to contact the server.